contrib/core-contract-tests/tests/pox-4/pox-4.stateful-prop.test.ts

```diff
@@ -0,0 +1,162 @@
+import { it } from "vitest";
+import { initSimnet } from "@hirosystems/clarinet-sdk";
+import { Real, Stub } from "./pox_CommandModel.ts";
+
+import {
+  getPublicKeyFromPrivate,
+  publicKeyToBtcAddress,
+} from "@stacks/encryption";
+import { StacksDevnet } from "@stacks/network";
+import {
+  createStacksPrivateKey,
+  getAddressFromPrivateKey,
+  TransactionVersion,
+} from "@stacks/transactions";
+import { StackingClient } from "@stacks/stacking";
+
+import fc from "fast-check";
+import { PoxCommands } from "./pox_Commands.ts";
+
+import fs from "fs";
+import path from "path";
+
+it("statefully interacts with PoX-4", async () => {
+  // SUT stands for "System Under Test".
+  const sut: Real = {
+    network: await initSimnet(),
```

Comment:
This could be causing the issue identified by @BowTiedRadone 
I'm currently investigating

Reply:
> Can we disable the implicit/global simnet object for this test? 

That would be doable by tweaking the setup file but I don't see a lot of advantage to do so.

> Or perhaps I can reference the implicit/global simnet instead of await initSimnet().

That would make sense yes 👍 It wouldn't fix the issue though, it wouldn't have much of an impact but I still recommend it

Reply:
OK, I can reference the implicit/global `simnet` instead of `await initSimnet()`.